reactive-app/src/main/resources/application.properties

```diff
@@ -9,3 +9,8 @@ quarkus.hibernate-orm.database.generation=drop-and-create
 # Reactive config
 quarkus.datasource.reactive.url=vertx-reactive:sqlserver://localhost:1433
 quarkus.datasource.reactive.max-size=200
+
+mp.context.ThreadContext.propagated=None
+mp.context.ThreadContext.cleared=None
+mp.context.ThreadContext.unchanged=Remaining
+quarkus.datasource.reactive.cache-prepared-statements=true
```

Comment:
This is not supported on the Reactive SQL Server Client

Reply:
ah, thanks!

cc/ @jamesfalkner

Reply:
Yeah, when I run this I get:

```
__  ____  __  _____   ___  __ ____  ______
 --/ __ \/ / / / _ | / _ \/ //_/ / / / __/
 -/ /_/ / /_/ / __ |/ , _/ ,< / /_/ /\ \
--\___\_\____/_/ |_/_/|_/_/|_|\____/___/
2021-11-22 13:55:44,392 WARN  [io.qua.config] (main) Unrecognized configuration key "quarkus.datasource.jdbc.url" was provided; it will be ignored; verify that the dependency extension for this configuration is set or that you did not make a typo
2021-11-22 13:55:46,306 INFO  [org.hib.rea.pro.imp.ReactiveIntegrator] (JPA Startup Thread: default-reactive) HR000001: Hibernate Reactive Preview
2021-11-22 13:55:47,821 INFO  [io.quarkus] (main) hibernate-reactive-panache-quickstart 1.0.0-SNAPSHOT on JVM (powered by Quarkus 2.4.2.Final) started in 4.390s. Listening on: http://0.0.0.0:8080
2021-11-22 13:55:47,822 INFO  [io.quarkus] (main) Profile prod activated.
2021-11-22 13:55:47,822 INFO  [io.quarkus] (main) Installed features: [cdi, hibernate-orm, hibernate-reactive, hibernate-reactive-panache, reactive-mssql-client, resteasy-reactive, resteasy-reactive-jackson, smallrye-context-propagation, vertx]
2021-11-22 13:55:49,227 ERROR [io.qua.ver.htt.run.QuarkusErrorHandler] (vert.x-eventloop-thread-3) HTTP Request to /fruits failed, error id: 85802e5d-f5e6-4ae7-b00d-c0ebf6d706b1-1: javax.enterprise.context.ContextNotActiveException
	at io.quarkus.arc.impl.ClientProxies.getDelegate(ClientProxies.java:40)
	at io.quarkus.hibernate.reactive.runtime.ReactiveSessionProducer_ProducerMethod_createMutinySession_1321d110ee9e92bda147899150401e0a136779c7_ClientProxy.arc$delegate(ReactiveSessionProducer_ProducerMethod_createMutinySession_1321d110ee9e92bda147899150401e0a136779c7_ClientProxy.zig:43)
	at io.quarkus.hibernate.reactive.runtime.ReactiveSessionProducer_ProducerMethod_createMutinySession_1321d110ee9e92bda147899150401e0a136779c7_ClientProxy.contains(ReactiveSessionProducer_ProducerMethod_createMutinySession_1321d110ee9e92bda147899150401e0a136779c7_ClientProxy.zig:630)
	at io.quarkus.hibernate.reactive.panache.common.runtime.AbstractJpaOperations.lambda$persist$1(AbstractJpaOperations.java:76)
	at io.smallrye.mutiny.operators.uni.UniOnItemTransformToUni$UniOnItemTransformToUniProcessor.performInnerSubscription(UniOnItemTransformToUni.java:68)
	at io.smallrye.mutiny.operators.uni.UniOnItemTransformToUni$UniOnItemTransformToUniProcessor.onItem(UniOnItemTransformToUni.java:57)
	at io.smallrye.mutiny.operators.uni.builders.UniCreateFromKnownItem$KnownItemSubscription.forward(UniCreateFromKnownItem.java:38)
	at io.smallrye.mutiny.operators.uni.builders.UniCreateFromKnownItem$KnownItemSubscription.access$100(UniCreateFromKnownItem.java:26)
```